AOBJ/type/zif_aff_aobj_v1.intf.abap

```diff
@@ -0,0 +1,74 @@
+INTERFACE zif_aff_aobj_v1
+
+  PUBLIC.
+
+  TYPES: ty_write_class TYPE c LENGTH 40.
+  TYPES: ty_delete_class TYPE c LENGTH 40.
+  TYPES: ty_reload_class TYPE c LENGTH 40.
```

Comment:
Do these types refer to classes? If so, maybe you just want to define one type. The maximum length of a class is 30 characters. 

Alternatively, maybe you want to reuse `zif_aff_types_v1=>ty_object_name_30`.